app-misc/openrgb-plugin-skin/openrgb-plugin-skin-0.9-r1.ebuild

```diff
@@ -17,6 +17,7 @@ KEYWORDS="amd64"
 
 RDEPEND="
 	>=app-misc/openrgb-0.9:=[-qt6(-)]
```

Comment:
It looks like this USEdep could also be cleaned up, now that the transition will not involve an IUSE qt6 (introduced in 815f98cf475127787bd3c45a4aedd4c0febea049 for transition) :)